### PR TITLE
Add sequence diagrams ( ForgetPassword and ProfileView)

### PR DESCRIPTION
**Pull request by tarassito**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=777)

## Code reviewers

- [x] @ashatna
- [ ] @ChristinaPastushchak
- [ ] @dotiurin
- [ ] @iryn34
- [ ] @tarassito
- [ ] @VasilHamulyak
- [ ] @VitalyiKhlivnyi
- [ ] @YurijH

### Second Level Review

- [ ] @ushevtsiv
- [x] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by VitalyiKhlivnyi — DISMISSED** [review]

**Review by ashatna — COMMENTED** [review]
на мою думку, треба уточнити, що таке ForgetPassword

**Review by ashatna — DISMISSED** [review]

File: documentation/sequence diagrams/authentication/Send email ForgetPassword Sequence Diagram.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+UI Component->+ForgetPassword: post(email)
+ForgetPassword->+Validators: updating_email_validate
+alt not updating_email_validate
+Validators->ForgetPassword:return False
+ForgetPassword-->UI Component: return RESPONSE_400_INVALID_DATA
+else:
+Validators-->-ForgetPassword : return True
+ForgetPassword->+CustomUser: get_by_email
+CustomUser-->-ForgetPassword: user_id
```
Comment (lhalam) [inline documentation/sequence diagrams/authentication/Send email ForgetPassword Sequence Diagram.txt]: returned CustomUser instance

File: documentation/sequence diagrams/authentication/Send email ForgetPassword Sequence Diagram.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ForgetPassword->+CustomUserMail: send_password_update_letter
```
Comment (lhalam) [inline documentation/sequence diagrams/authentication/Send email ForgetPassword Sequence Diagram.txt]: before add create token
send_password_update_letter(user, token)

File: documentation/sequence diagrams/authentication/Set new password Sequence Diagram.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+title  Set new password Sequence Diagram
+
+CustomUserMail-->+UI Component: get()
```
Comment (lhalam) [inline documentation/sequence diagrams/authentication/Set new password Sequence Diagram.txt]: ?token

File: documentation/sequence diagrams/authentication/Set new password Sequence Diagram.txt [hunk 4]
```diff
@@ -0,0 +1,15 @@
+title  Set new password Sequence Diagram
+
+CustomUserMail-->+UI Component: get()
+UI Component->+ForgetPassword:put(token)
```
Comment (lhalam) [inline documentation/sequence diagrams/authentication/Set new password Sequence Diagram.txt]: before adding input fields(user) and validate field(UIcomponent) logic

File: documentation/sequence diagrams/authentication/Set new password Sequence Diagram.txt [hunk 5]
```diff
@@ -0,0 +1,15 @@
+title  Set new password Sequence Diagram
+
+CustomUserMail-->+UI Component: get()
+UI Component->+ForgetPassword:put(token)
+ForgetPassword->+CustomUser: get_by_id
```
Comment (lhalam) [inline documentation/sequence diagrams/authentication/Set new password Sequence Diagram.txt]: before add identifier

File: documentation/sequence diagrams/authentication/Set new password Sequence Diagram.txt [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+CustomUserMail-->+UI Component: get()
+UI Component->+ForgetPassword:put(token)
+ForgetPassword->+CustomUser: get_by_id
+alt not user
+CustomUser-->ForgetPassword:return False
+ForgetPassword-->UI Component:RESPONSE_404_OBJECT_NOT_FOUND
+else:
+CustomUser-->-ForgetPassword: user_id
+ForgetPassword->+BD: data(new password)
```
Comment (lhalam) [inline documentation/sequence diagrams/authentication/Set new password Sequence Diagram.txt]: add updating_password_validate

**Review by lhalam — CHANGES_REQUESTED** [review]
To improve!

**Comment by tarassito** [comment]
improved

**Review by lhalam — DISMISSED** [review]

**Review by ChristinaPastushchak — APPROVED** [review]

**Pull request merged**